keps/sig-node/3619-supplemental-groups-policy/README.md

```diff
@@ -1039,8 +1119,21 @@ For each of them, fill in the following information by copying the below templat
     - Testing: Are there any tests for failure mode? If not, describe why.
 -->
 
+None.
```

Comment:
I think the "pod failing to schedule because the node doesn't support it" is a failure mode you can document here.

Reply:
Thanks. But, this section defines "What are OTHER known failure modes?", right? 

I think the failure mode "node does not support it" was clearly mentioned above. This proposes to raise a kubelet admission error for this case. 

Would you like to describe the failure mode here even though it was stated?